fastapi/utils.py

```diff
@@ -61,10 +61,11 @@ def create_cloned_field(field: Field) -> Field:
         use_type = create_model(
             original_type.__name__,
             __config__=original_type.__config__,
-            __validators__=original_type.__validators__,  # type: ignore
         )
         for f in original_type.__fields__.values():
```

Comment:
I wonder if you can't just do `use_type.__fields__ = original_type.__fields__` and be done with it?

Reply:
I don't believe create_model has that signature. I played around with some options, but nothing stuck.

However, feel free to discard my PR if there is a better solution, by all means!

Reply:
No I meant after the create_model call, instead of looping. Just replacing the whole dict instead; the call to create_model could stay as it was. I think that might pass mypy too.

Reply:
Updated, as suggested.